### PR TITLE
Improved knowledge Base

### PR DESCRIPTION
**Pull request by prachi237**
PR Checklist:

- [x] make all edits to the docs in the `src` directory, not in `docs`
- [x] note any issues closed by this PR with [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [x] put any other relevant information below

### CODE REVIEW

File: src/maintainer/knowledge_base.rst [hunk 1]
```diff
@@ -113,20 +113,18 @@ versions are:
 * For Python 2.7: Visual C++ 9.0
 * For Python 3.5–3.7: Visual C++ 14.0
 
-While you can obtain these tools by installing the right version of the full
+You can obtain these tools by installing the right version of the full
```
Comment (viniciusdc) [inline src/maintainer/knowledge_base.rst:116]:
```suggestion
While you can obtain these tools by installing the right version of the full
```

**Review by viniciusdc — COMMENTED** [review]
Lgtm

**Comment by prachi237** [comment]
@viniciusdc  done

File: src/maintainer/knowledge_base.rst [hunk 2]
```diff
@@ -232,10 +230,13 @@ Core Dependency Tree Packages (CDTs)
...
-However, there are a few exceptions: some dependencies are so close to the system
-that they are not packaged with ``conda-forge``. These dependencies have to be satisfied with
-*Core Dependency Tree* (CDT) packages. A CDT package consists of repackaged CentOS binaries from the
-appropriate version, either 6 or 7 depending on user choice and platform. We manage the build of CDT
+However, there are a few exceptions: 
+
+Some dependencies are so close to the system that they are not packaged with ``conda-forge``. 
+These dependencies have to be satisfied with *Core Dependency Tree* (CDT) packages. 
+
+A CDT package consists of repackaged CentOS binaries from the appropriate version,
```
Comment (viniciusdc) [inline src/maintainer/knowledge_base.rst:238]: I think we could remove this definition (CDT) here and link directly to the glosary https://conda-forge.org/docs/misc/00_intro.html#glossary, what do you think ?

**Review by viniciusdc — COMMENTED** [review]
Lgtm

File: src/maintainer/knowledge_base.rst (hunk 2 above)
Comment (prachi237) [reply src/maintainer/knowledge_base.rst:238]: from line number 238 right...yeah great will do that

Comment (prachi237) [reply src/maintainer/knowledge_base.rst:238]: > I think we could remove this definition (CDT) here and link directly to the glosary https://conda-forge.org/docs/misc/00_intro.html#glossary, what do you think ?

btw shall I do that

Comment (beckermr) [reply src/maintainer/knowledge_base.rst:238]: Please don't as requested by Isuru

**Comment by prachi237** [comment]
@viniciusdc  please review

**Review by viniciusdc — APPROVED** [review]
lgtm

**Pull request merged**